middle_end/flambda2/simplify/unboxing/unbox_continuation_params.ml

```diff
@@ -18,31 +18,39 @@ open! Simplify_import
 module U = Unboxing_types
 module Decisions = U.Decisions
 
-let refine_decision_based_on_arg_types_at_uses ~pass ~rewrite_ids_seen nth_arg
-    arg_type_by_use_id (decision : U.decision) =
+let refine_decision_based_on_arg_types_at_uses ~pass ~rewrite_ids_seen
+    ~rewrite_ids_invalid nth_arg arg_type_by_use_id (decision : U.decision) =
```

Comment:
rather `invalid_rewrite_ids` instead ?

Reply:
Well not really it's not the id that is invalid...

Reply:
yeah, it's hard to find a fitting name for that one, basically its the set of rewrite ids whose apply_cont are/will be turned into invalid (because trying to compute one of the arguments for that apply cont  resulted in an invalid)

Reply:
I've done the renaming